lang/cs/Org.Apache.REEF.Driver/Bridge/ClrHandlerHelper.cs

```diff
@@ -35,18 +34,41 @@ namespace Org.Apache.REEF.Driver.Bridge
 {
     public class ClrHandlerHelper
     {
+        private const string DllExtension = ".dll";
         private static readonly Logger LOGGER = Logger.GetLogger(typeof(ClrHandlerHelper));
 
+        /// <summary>
+        /// The set of REEF assemblies required for the Driver.
+        /// </summary>
         public static string[] ReefAssemblies
```

Comment:
Is this used anywhere? If so, it feels like we should define it as `internal` there, as applications shouldn't have to know this.